doc/glossary.org

```diff
@@ -1,8 +1,12 @@
 #+title: Glossary
 
-J/Link - Integrates the Wolfram Language and Java.
+J/Link - The bridge that integrates the Wolfram Language and Java.
 
 MathLink - The communication protocol binary that allows external programs to call Mathematica and vice versa.
 
 LoopbackLink - A special type of link that has both ends connected to the same program. Used for temporarily holding expressions that are being moved between links. Much of the utility of LoopbackLinks to users of the MathLink API is redundant when compared to J/Link's Expr class.
 (https://reference.wolfram.com/language/JLink/ref/java/com/wolfram/jlink/LoopbackLink.html)
+
+Context - The Wolfram equivalent of a namespace.
+
+Package - A Wolfram file (extension .m or .wl) that defines/contains a namespace and the interface to said namespace.
```

Comment:
```suggestion
Package - A Wolfram file (extension .m or .wl) that defines/contains a Context and the interface to said context.
```
isn't this more accurate and consistent?

Reply:
It is more accurate; I think I must have been trying to put it in terms that a clojurist would understand.